_apidocs/get-opportunities-public-api.md

```diff
@@ -216,6 +217,9 @@ Note: Response for one record is provided as an example <br>
             "typeOfSetAside": "7",
             "responseDeadLine": null,
             "naicsCode": "236220",
+			"naicsCodes": [
```

Comment:
There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?

Reply:
Resolved